content/docs/06.enterprise/01.overview/standalone-server-installation.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Standalone Server Installation Guide (Enterprise Edition)
+icon: /docs/icons/installation.svg
+editions: ["EE"]
+---
+
+Install Kestra on a standalone server with a simple executable file.
+
+# Standalone Server Installation Guide (Enterprise Edition)
+
+To deploy Kestra without Docker, there's a standalone JAR available that allows deployment in any environment that has JVM version 21+. 
+
+# Instructions
+
+The following is a quick start guide to get your Kestra Enterprise Edition up and running in standalone mode.
+
+---
+
+## Standalone JAR
+
+Download the latest version of the Kestra EE JAR from:
+
+[http://registry.kestra.io/exe/latest](http://registry.kestra.io/exe/latest)
+
+**Credentials:**
+
+- **Username**: `license-id`  
+- **Password**: `fingerprint`
+
+This provides a single JAR file that can be used to start Kestra. Store the file in your execution environment as `kestra` (make it executable).
+
+To make the file executable, Linux or MacOS users use the following with filename:
+
+```bash
+chmod +x kestra.jar
+```
+
+Or with a file path:
+
+```bash
+mv kestra.jar /usr/local/bin/kestra # Replace with your execution environment file path
+chmod +x /usr/local/bin/kestra
+```
+
+For Windows users:
+
+```powershell
+java -jar kestra.jar
+```
+
+Or with a file path assuming execution from the current directory:
+
+```powershell
+java -jar kestra.jar server standalone -c ./application.yml -p ./plugins --port=8080
+```
+
```

Comment:
After checking, it downloads a zip file called `latest.zip` which contains the executable in a format like this: `kestra-ee-0.22.26` which can be executed with `./kestra-ee-0.22.26 server local` for example. Does need config for license id and key.

`chmod` command may be required I guess. Also potentially add clarification that the name contains `-ee-VERSION`.